<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>635</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenMW Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../files/launcher/launcher.qrc">
    <normaloff>:/images/openmw.png</normaloff>:/images/openmw.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="pagesWidget"/>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>OpenMW version</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="dataAction"/>
   <addaction name="graphicsAction"/>
   <addaction name="settingsAction"/>
   <addaction name="importAction"/>
  </widget>
  <action name="dataAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../files/launcher/launcher.qrc">
     <normaloff>:/images/openmw-plugin.png</normaloff>:/images/openmw-plugin.png</iconset>
   </property>
   <property name="text">
    <string>Data Files</string>
   </property>
   <property name="toolTip">
    <string>Allows to setup data files and directories</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="graphicsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../files/launcher/launcher.qrc">
     <normaloff>:/images/preferences-video.png</normaloff>:/images/preferences-video.png</iconset>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="toolTip">
    <string>Allows to change display settings</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../files/launcher/launcher.qrc">
     <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Allows to tweak engine settings</string>
   </property>
  </action>
  <action name="importAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../files/launcher/launcher.qrc">
     <normaloff>:/images/preferences-advanced.png</normaloff>:/images/preferences-advanced.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Allows to import data from original engine</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../files/launcher/launcher.qrc"/>
 </resources>
 <connections/>
</ui>
